src/Controller/TreeController.php

```diff
@@ -102,6 +102,10 @@ public function reorderAction(Request $request)
             return new JsonResponse(array('Parameters parent, dropped and target has to be set to reorder.'), Response::HTTP_BAD_REQUEST);
         }
 
+        if (in_array($position, array('over', 'child'))) {
+            return new JsonResponse(array('Can not resort into a collection.'), Response::HTTP_BAD_REQUEST);
```

Comment:
i mean "reorder", and that should be done, when position is `before` or `after` a node. Reorder will interally set the position property of a node collection.